src/main/java/org/jasig/portlet/widget/links/ResourceLinkService.java

```diff
@@ -0,0 +1,89 @@
+package org.jasig.portlet.widget.links;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ResourceLinkService {
+
+    private static final Logger log = LoggerFactory.getLogger(ResourceLinkService.class);
```

Comment:
📓 Lombok provides a nice helper to normalize log usage
Ref: https://projectlombok.org/features/log